ilastik/plugins_default/tracking_mamut_export.py

```diff
@@ -38,15 +38,16 @@ def checkFilesExist(self, filename):
         ''' Check whether the files we want to export are already present '''
         return os.path.exists(filename + '_mamut.xml') or os.path.exists(filename + '_bdv.xml') or os.path.exists(filename + '_raw.h5')
 
-    def export(self, filename, hypothesesGraph, *, objectFeaturesSlot, bdvFilepathSlot, **kwargs):
+    def export(self, filename, hypothesesGraph, *, objectFeaturesSlot, additionalPluginArgumentsSlot, **kwargs):
```

Comment:
do we still want to support kwargs now, since you added the slot? Or is some more refactoring needed in order to get rid of it?